tinytuya/BulbDevice.py

```diff
@@ -12,10 +12,10 @@
 
  Functions
     BulbDevice Class methods
-        rgb_to_hexvalue(r, g, b, use_rgb=False):
-        hsv_to_hexvalue(h, s, v, use_rgb=False):
-        hexvalue_to_rgb(hexvalue, use_rgb=None):
-        hexvalue_to_hsv(hexvalue, use_rgb=None):
+        rgb_to_hexvalue(r, g, b, hex_format):
+        hsv_to_hexvalue(h, s, v, hex_format):
+        hexvalue_to_rgb(hexvalue, hex_format=None):
+        hexvalue_to_hsv(hexvalue, hex_format=None):
```

Comment:
Spotted one thing in docs: change to hexformat instead of hex_format

Reply:
Yep, noticed right after committing.  Fixed.